hack/tools/internal/tilt-prepare/main.go

```diff
@@ -228,9 +226,6 @@ func setTiltSettingsDefaults(ts *tiltSettings) {
 		if p.ProfilerPort == nil {
 			p.ProfilerPort = pointer.Int(0)
 		}
```

Comment:
Shouldn't we also drop this part then?

Reply:
Also:

Is/was ProfilerPort buggy?

https://github.com/kubernetes-sigs/cluster-api/blob/0a7a879d06631e6073b63a5d06695d779a680e60/hack/tools/internal/tilt-prepare/main.go#L805-L807

This is hardcoded to 6060, while the property is the int which should set the port?!

Reply:
ProfilerPort is the port on your local machine. 6060 the one of the server in the Pod.

My idea was to keep this around because it's a lot easier to get profiles via an unprotected port if necessary.

Basically parca gets profiles via the secure port (and verifies it this way) and if you want to get profiles manually you can use this port

Reply:
Pushed a commit to always add the parca annotation